spec/requests/users_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    before(:each) { get users_path } # get(:users)
+
+    it 'returns response status success' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'renders template success' do
+      expect(response.body).to match(/Frank/)
+    end
+
+    it 'shows placeholder text success' do
+      expect(response.body).to include('Frank')
+    end
+  end
+end
```

Comment:
- [ ] according to the requirements the index and show actions for User and Post controllers should be tested, however, you are only testing the index actions
- [ ] Please remember that the Spec file should include tests that ensures correct view template is rendered